### ISSUE TITLE
Help for the final phase of JS modernization

### ISSUE THREAD

**Issue by slim16165** [krabina/Semantic-MediaWiki-Graph#3]
Hi, 
I've rewrote a lot of parts to use OOP and the latest version of D3 and Select2, making it modular, following the SRP and I hope easier to maintain. However, I am not familiar with SVG and D3 library, and I am having a hard time debugging the graphics. Would you be able to take a look at it when you have time?

**Comment by krabina** [comment]
Sorry, I did not write this extension and I am not a developer. In can help testing some things as soon as they are running in MW 1.35 (which I am currenltly using), but will not be able to help you with coding.

**Comment by krabina** [comment]
Maybe @semantisch can help? - It is about JavaScript

**Comment by slim16165** [comment]
Yes, the nodes are drawed correctly, while the edges and the legend still
have some issue.
